students/17th/team4/leejunha/user/views.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django.views import View
+from django.http  import JsonResponse
+
+from .models      import Account
+
+class UserSignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            email        = data['email']
+            password     = data['password']
+            user_name    = data['user_name']
+            phone_number = data['phone_number']         
+
+            if len(password) < 8:
```

Comment:
```suggestion
            if len(password) <  MINIMUM_PASSWORD_LENGTH:
```
비교값이 되는 8과 같은 값은, 상수로 지정해서 관리하는게 좋습니다.